frontend/src/lib/types/index.d.ts

```diff
@@ -350,6 +350,21 @@ export declare namespace fwd {
     WriteTo(w?: unknown): [number, Error];
   }
 }
+export declare namespace mod {
```

Comment:
These came in automatically when the typings got regenerated to pick up the updated calculate signature. I wasn't sure if you'd prefer for me to leave them in vs omit them from the PR, I'm fine with whichever you prefer.